celery_sqlalchemy_scheduler/db/models.py

```diff
@@ -1,20 +1,22 @@
 import datetime as dt
-from typing import Dict, Union
+from typing import Any, Dict, Union
 
 import pytz
 import sqlalchemy as sa
 from celery import schedules
 from celery.utils.log import get_logger
-from sqlalchemy import func
+from sqlalchemy import MetaData, func
 from sqlalchemy.engine import Engine
 from sqlalchemy.event import listen
+from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import Session, class_mapper, foreign, relationship, remote
 from sqlalchemy.sql import insert, select, update
 
-from .session import ModelBase
-from .tzcrontab import TzAwareCrontab
+from celery_sqlalchemy_scheduler.tzcrontab import TzAwareCrontab
 
-logger = get_logger("celery_sqlalchemy_scheduler.models")
+logger = get_logger("celery_sqlalchemy_scheduler.db.models")
```

Comment:
Provides the module name automatically

```suggestion
logger = logging.getLogger(__name__)
```